sci-libs/spyking-circus/spyking-circus-0.9.9.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Fast spike sorting by template matching"
+HOMEPAGE="https://github.com/spyking-circus/spyking-circus/"
+SRC_URI="https://github.com/spyking-circus/spyking-circus/archive/${PV}.tar.gz"
+LICENSE="CeCILL-2"
+
+SLOT="0"
+KEYWORDS="~amd64 ~amd64-linux ~x86 ~x86-linux"
+
+RDEPEND="
+	>=dev-python/blosc-1.8[${PYTHON_USEDEP}]
+	>=dev-python/colorama-0.4.1[${PYTHON_USEDEP}]
+	>=dev-python/cython-0.29.14[${PYTHON_USEDEP}]
+	>=dev-python/h5py-2.9.0[${PYTHON_USEDEP}]
+	>=dev-python/matplotlib-2.2.4[${PYTHON_USEDEP}]
+	>=dev-python/mpi4py-3.0.0[${PYTHON_USEDEP}]
+	>=dev-python/numpy-1.17.4[${PYTHON_USEDEP}]
+	>=dev-python/pandas-0.21[${PYTHON_USEDEP}]
+	>=dev-python/psutil-5.6.7[${PYTHON_USEDEP}]
+	>=dev-python/statsmodels-0.10.1[${PYTHON_USEDEP}]
+	>=dev-python/tqdm-4.40.0[${PYTHON_USEDEP}]
+	>=dev-python/scipy-1.3.1[${PYTHON_USEDEP}]
+	>=sci-libs/scikits_learn-0.21.3
```

Comment:
New name of dependency is `sci-libs/scikit-learn`

Reply:
Changed :)